modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -10,3 +10,5 @@ Review the following limitations before installing and using the AWS Load Balanc
 * The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller. As a result of disabling the IP traffic mode, the AWS Load Balancer Controller cannot use the pod readiness gate.
 
 * The AWS Load Balancer Operator adds command-line flags such as `--disable-ingress-class-annotation` and `--disable-ingress-group-name-annotation` to the AWS Load Balancer Controller. Therefore, the AWS Load Balancer Operator does not allow using the `kubernetes.io/ingress.class` and `alb.ingress.kubernetes.io/group.name` annotations in the `Ingress` resource.
+
+* The AWS Load Balancer Operator must be configured so the SVC is "NodePort" type instead of "ClusterIP" type.
```

Comment:
Please replace "ClusterIP" with "LoadBalancer", so it can be:
`The AWS Load Balancer Operator must be configured so the SVC is "NodePort" type instead of "LoadBalancer" type.`

Reply:
Of course, yes. Looks like this might need to be included in the install documentation as well?

Reply:
Thanks for raising this PR, Jessi. No installation doc update should be needed as the Networking docs don't really connect to one one platform.

Reply:
@dfitzmau My apologies, I will need to add this as a consideration in the AWS Load Balancer install or post-install configuration documentation.